libafl/src/common/nautilus/grammartec/tree.rs

```diff
@@ -346,7 +360,16 @@ impl Tree {
         ctx: &Context,
     ) {
         match ctx.get_rule(ruleid) {
-            Rule::Plain(..) | Rule::Script(..) => {
+            Rule::Plain(..) => {
```

Comment:
Can move these into a method so we don't have to copy&paste?

Reply:
Oh yeah. That's right.